magento/magento2-login-as-customer
# 17 notification banner

Notification banner on storefront [LoginAsCustomer]

### Description (*)
Because merchant admin users can make destructive and significant changes to user accounts when logged in as customers, it's important to provide persistent notification in the browser of which customer account they are viewing. This pull request adds this persistent notification.


### Fixed Issues (if relevant)
1. magento/magento2-login-as-customer#17:Notification banner on storefront

### Manual testing scenarios (*)
As a merchant admin user, log in to a customer account using Login as Customer
1.Verify banner is displayed based on approved UX design
2.Verify correct message is displayed based on approved message from Technical Writer
3.Take additional actions in storefront:
3.1 View product detail page
3.2 View category page
3.3. View wishlist
3.4. View cart
3.5. Add item to cart
3.6. Place order
4.Verify banner is persistent and appears during all page views in session
5. If visit the website as a guest, or login using regular login form, then you will should not see this message.


### Questions or comments
Need approved UX design and message. Right now it use the same design as demo notice (https://docs.magento.com/m2/ce/user_guide/design/demo-notice.html) and text:

Attention! You are logged in as the customer.

### Contribution checklist (*)
 - [+] Pull request has a meaningful description of its purpose
 - [+] All commits are accompanied by meaningful commit messages
